ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -1,43 +1,126 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test environment API - TestEnv\n",
+    "\n",
+    "The test environment is primarely defined by the target configuration (see <b>conf</b> below).\n",
```

Comment:
BTW, not sure if you wrote in the `<b>`s or ipython did that but in markdown you can just use `**` e.g. `**important**` -> **important**